drake/examples/Pendulum/Pendulum.h

```diff
@@ -0,0 +1,41 @@
+#ifndef _PENDULUM_H_
+#define _PENDULUM_H_
+
+#include <iostream>
+#include "DrakeSystem.h"
+
+using namespace std;
+using namespace Eigen;
```

Comment:
Shall we put all the `using namespace` in the cpp file? Otherwise any file that include `Pendulum.h` will be forced to use standard and Eigen namespaces.